### PR TITLE
Da scriptless scripts

### PR DESCRIPTION
**Pull request by anselld**
Hi Kevoulee
Please could you review my formatting changes to 'Scriptless Scripts' and let me know if you have any questions.
Please could you help with the following:
1. For consistency, I removed the underlines between sections. The other reports I have formatted to date did not have them. If this is a TLU requirement, please let me know, as I will need to go back and change the others.
2. The link to [6] does not exist.
3. In line 22, the reference to #5 Scriptless Scripts is unclear. Please help.
Thanks and regards
Denise

### CODE REVIEW

**Comment by Kevoulee** [comment]
Thank you for the review @anselld. I wasn't sure how to phrase line 22. Hansie had previously introduced scriptless scripts in his layer 2 scaling report and I wanted to provide a link to his work.

**Comment by anselld** [comment]
Hi Kevoulee. Sorry for the false alarm: the link to [6] is fine.
The link to Hansie's report is fine if the #5 is deleted.
Please let me know if you are happy with the formatting before I start on the editing.

**Comment by Kevoulee** [comment]
Hi @anselld, perfect, I am happy with the formatting. Thank you so much

**Comment by anselld** [comment]
Pleasure, Kevoulee.

**Comment by anselld** [comment]
Hi Kevoulee
I've completed the editing of Scriptless Scripts and have pushed and committed the file. It looks good.
Please could you review the suggested changes and let me have your comments.
One question, please: Should Scriptless Scripts be in intitial caps, as edited in the text? I am making a list to include in TLU's Style Guide, Appendix B - Conventions.
Thanks!

**Comment by Kevoulee** [comment]
Hi @anselld, thank you for editing my work. Scriptless Scripts be in initial caps- thank you for changing in the report. Please let me about my mistake with x and r in italics. Thank you again- this is perfect.

File: src/cryptography/scriptless-scripts/introduction-to-scriptless-scripts.md [hunk 1]
```diff
@@ -82,53 +89,53 @@ $$
...
 
-The position on the line chosen is taken as the hash of all the data that one needs to commit to, the digital signature. The verification equation involves the multiplication of each of the terms in the equation by G and takes in account the cryptographic assumption (discrete log) where G can be multiplied in but not divided out, thus preventing deciphering. 
+The position on the line chosen is taken as the hash of all the data that one needs to commit to, the digital signature. The verification equation involves the multiplication of each of the terms in the equation by G and takes into account the cryptographic assumption (discrete log) where G can be multiplied in but not divided out, thus preventing deciphering. 
 
 $$
 sG=rG+exG
 $$
 
-ECDSA signatures (used in Bitcoin) are not linear in *x* and *r*, and thus less useful. [[2]] 
+Elliptic Curve Digital Signature Algorithm (ECDSA) signatures (used in Bitcoin) are not linear in x and r, and thus less useful. [[2]]
```
Comment (anselld) [inline src/cryptography/scriptless-scripts/introduction-to-scriptless-scripts.md]: Hi Kevoulee, sorry - x and r should be italics - thanks for pointing that out. I'll fix it.

**Comment by Kevoulee** [comment]
@hansieodendaal, so sorry to bother you, would it be alright if I squashed and merged- I don't want to jump the gun

**Comment by hansieodendaal** [comment]
@Kevoulee, please go ahead. All good as I can see the process was followed perfectly.

**Pull request merged**